04-oop-1-class-constructor/index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "4 - OOP - Objects, Classes & Constructor"
-nav_order: 4
+title: "04 - OOP - Objects, Classes & Constructor"
+nav_order: 5
 has_children: false
-nav_exclude: true
+nav_exclude: false
```

Comment:
It's ok to let it visible, but then please just merge this PR moments before the class